Bottom: bcf7d3396e57190d8d8fb3b0bae86dd5819e0668
Top:    c55f5f5803684d0accb3987c1fe185a8da341f7a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-26 16:13:04 +0000

WAS: Use motion hints when processing pointer motion over the drawing area

Not sure what the remaining changes here are for!


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 46a7380..66bfa74 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -761,7 +761,7 @@ ghid_show_crosshair (gboolean paint_new_location)
 
   glBegin (GL_LINES);
 
-#if 1
+#if 0
   if (x_prev >= 0 && !paint_new_location)
     draw_crosshair (x_prev, y_prev);
 #endif
@@ -774,7 +774,7 @@ ghid_show_crosshair (gboolean paint_new_location)
 
   glBegin (GL_QUADS);
 
-#if 1
+#if 0
   if (x_prev >= 0 && draw_markers_prev & !paint_new_location)
     {
       glVertex2i (0,                  y_prev - VCD);
